src/confluent_kafka/admin/__init__.py

```diff
@@ -87,28 +83,37 @@ def __str__(self):
 @functools.total_ordering
 class ConfigResource(object):
     """
-    Class representing resources that have configs.
-
-    Instantiate with a resource type and a resource name.
+    Represents a resource that has configuration, and (optionally)
+    a collection of configuration values for that resource. Used by
+    describe_configs() and alter_configs().
+
+    Parameters
+    ----------
+    restype : `ConfigResource.Type`
+       The resource type.
+    name : `str`
+       The resource name, which depends on the resource type. For RESOURCE_BROKER, the resource name is the broker id.
+    set_config : `dict`
+        The configuration to set/overwrite. Dictionary of str, str.
     """
 
     class Type(Enum):
         """
-        ConfigResource.Type depicts the type of a Kafka resource.
+        Enumerates the different types of a Kafka resources.
         """
         UNKNOWN = RESOURCE_UNKNOWN  #: Resource type is not known or not set.
         ANY = RESOURCE_ANY  #: Match any resource, used for lookups.
-        TOPIC = RESOURCE_TOPIC  #: Topic resource. Resource name is topic name
-        GROUP = RESOURCE_GROUP  #: Group resource. Resource name is group.id
-        BROKER = RESOURCE_BROKER  #: Broker resource. Resource name is broker id
+        TOPIC = RESOURCE_TOPIC  #: Topic resource. Resource name is topic name.
+        GROUP = RESOURCE_GROUP  #: Group resource. Resource name is group.id.
+        BROKER = RESOURCE_BROKER  #: Broker resource. Resource name is broker id.
 
     def __init__(self, restype, name,
                  set_config=None, described_configs=None, error=None):
         """
         :param ConfigResource.Type restype: Resource type.
-        :param str name: Resource name, depending on restype.
-                          For RESOURCE_BROKER the resource name is the broker id.
-        :param dict set_config: Configuration to set/overwrite. Dict of str, str.
+        :param str name: The resource name, which depends on restype.
```

Comment:
Adding "The" doesn't really add anything here (and elsewhere), and is added inconsistently.